### PR TITLE
doc: release-notes-latest: changes in FS and stream_flash from upmerge 20210305

### PR DESCRIPTION
**Pull request by nvlsianpu**
Added notes on changes in flash driver, FS, stream_flash.

Signed-off-by: Andrzej Puzdrowski <andrzej.puzdrowski@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/releases/release-notes-latest.rst [hunk 1]
```diff
@@ -129,7 +129,40 @@ The current |NCS| release is based on Zephyr v2.5.99.
...
+    * SoC embedded flash driver enabling imposes selection of
+      :option:`CONFIG_MPU_ALLOW_FLASH_WRITE` if ARM MPU enabled.
+
+
+* Libraries / Subsystems:
+
+  * File systems:
+
+    * Added c:func:`fs_file_t_init` function for initialization of
+      c:type:`fs_file_t` objects. Any :c:type:`fs_file_t` objects must now be
```
Comment (de-nordic) [inline doc/nrf/releases/release-notes-latest.rst:145]: maybe "must be now (...)" -> "require initialization, by calling the function, prior to their first use."

**Review by de-nordic — APPROVED** [review]
Left note, but generally I think this is OK.

**Review by ru-fu — CHANGES_REQUESTED** [review]
I'll add a commit.

File: doc/nrf/releases/release-notes-latest.rst [hunk 2]
```diff
@@ -129,7 +129,40 @@ The current |NCS| release is based on Zephyr v2.5.99.
 
 The following list summarizes the most important changes inherited from upstream Zephyr:
 
-* No changes yet
+* Drivers:
+
+  * Flash
+
+    * SoC embedded flash driver enabling imposes selection of
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:136]: I'll remove this one - it is covered by https://github.com/nrfconnect/sdk-nrf/pull/4353/

**Comment by nvlsianpu** [comment]
@ru-fu applied

**Review by ru-fu — APPROVED** [review]

**Comment by nvlsianpu** [comment]
@mbolivar-nordic Can you tell why OSS history check fails on such a trivial patch?
https://github.com/nrfconnect/sdk-nrf/pull/4364/checks?check_run_id=2401682824

**Comment by nvlsianpu** [comment]
^^ I think I forgot to rebase the patch.

**Pull request merged**